app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -175,38 +268,17 @@ def handle(self, *args, **options):
             if not b._val_usd_db:
                 continue
 
-            index_terms = []
-            if not should_suppress_leaderboard(b.bounty_owner_github_username):
-                index_terms.append(b.bounty_owner_github_username)
-                if b.org_name:
-                    index_terms.append(b.org_name)
-            for fulfiller in b.fulfillments.filter(accepted=True):
-                if not should_suppress_leaderboard(fulfiller.fulfiller_github_username):
-                    index_terms.append(fulfiller.fulfiller_github_username)
-            for keyword in b.keywords_list:
-                keyword = keyword.lower()
-                index_terms.append(keyword)
-
-            index_terms.append(b.token_name)
-
+            index_terms = bounty_index_terms(b)
             sum_bounties(b, index_terms)
 
-        # tips
+        # get tips
         tips = Tip.objects.exclude(txid='').filter(network='mainnet')
```

Comment:
As mentioned in your original comment, maybe we should define `network` at entry based on whether or not `DEBUG` is `True`.

`network = 'rinkeby' if settings.DEBUG else 'mainnet'` and use that throughout the file? 🤔

Reply:
Good idea....that makes sense.